index.js

```diff
@@ -42,38 +43,26 @@ function all(tree, file, config, next) {
      * @param {NLCSTNode} parent - `parent` of `node`.
      */
     function one(node, index, parent) {
+        var isCorrect = true;
+        var word = toString(node);
+
+        if (includes(ignore, word)) return;
```

Comment:
Please use the lint tools provided, this does not adhere to the style for this project!